docs/scripts/clean.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
 rm -rf 'processed-docs' 'processed-docs-cache'
-rm -rf  docs/reference/aztecjs 'docs/reference/smart_contract_reference/aztec-nr'
+rm -rf  docs/reference/developer_references/aztecjs 'docs/reference/developer_references/smart_contract_reference/aztec-nr'
```

Comment:
Could have all paths in 'single quotes'. Not sure why the first path wasn't in quotes all along.

Reply:
done